test/examples/ieee14bus.jl

```diff
@@ -0,0 +1,69 @@
+using Test: @test
+using PowerDynamics
+using OrderedCollections: OrderedDict
+
+buses=OrderedDict(
```

Comment:
@antonplietzsch It would be handy to include the code from examples/ieee14bus/buildexample.jl here simply instead of defining the IEEE14 bus system two times in the repo?

Reply:
Yes, good point. I'll do that.